subtests/docker_cli/build/build.py

```diff
@@ -64,212 +72,243 @@ def setup(self):
             os.write(busybox, data)
             os.close(busybox)
 
-        for filename in self.config['source_dirs'].split(','):
+        source_dirs = self.config['source_dirs']
+        for dir_path in get_as_list(source_dirs):
             # bindir is location of this module
-            src = os.path.join(self.bindir, filename)
+            src = os.path.join(self.bindir, dir_path)
             # srcdir is recreated if doesn't exist or if test version changes
-            dst = os.path.join(self.srcdir, filename)
+            dst = os.path.join(self.srcdir, dir_path)
             shutil.copytree(src, dst)
-            # copy the busybox
+            # copy busybox
             shutil.copy(os.path.join(self.srcdir, 'busybox'),
                         os.path.join(dst, 'busybox'))
 
-    def initialize(self):
-        super(build, self).initialize()
-        # Most tests use 'empty_base_image'. Add it only here
-        tarball = open(os.path.join(self.bindir, 'empty_base_image.tar'), 'rb')
-        dkrcmd = DockerCmd(self, 'import', ["-", "empty_base_image"])
-        mustpass(dkrcmd.execute(stdin=tarball))
-
-    def cleanup(self):
-        super(build, self).cleanup()
-        DockerImages(self).remove_image_by_full_name("empty_base_image")
-
 
 class build_base(subtest.SubSubtest):
 
-    def dockerfile_path(self, path):
-        if path[0] == '/':
+    def dockerfile_dir_path(self, dir_path):
+        if isinstance(dir_path, basestring) or dir_path is None:
```

Comment:
> The isinstance() will return true normally (I'm not try with other value, just dir_path = "/full"), is it excepted for some 
> reseaons? Or should we change to "not isinstance()" to avoid go into this if....

Woops, you're correct, it should be ``not isinstance(...)``, I'll fix it...